ui/components/multichain/pages/send/components/recipient.tsx

```diff
@@ -58,15 +58,16 @@ export const SendPageRecipient = () => {
   const recipient = useSelector(getRecipient);
   const userInput = useSelector(getRecipientUserInput) || '';
 
-  const domainResolution = useSelector(getDomainResolution);
+  const domainResolutions = useSelector(getDomainResolutions) || [];
   const domainError = useSelector(getDomainError);
   const domainWarning = useSelector(getDomainWarning);
 
   let addressBookEntryName = '';
+  const domainResolution = domainResolutions[0];
```

Comment:
? 🤔

Reply:
Hmm, missed this component being used anywhere before so I was updating it so tests would pass. Idk how I should proceed here tbh because the design they use for their `AddressListItem` component is divergent from ours.

Reply:
Perhaps we need to loop in @darkwing